utils.go

```diff
@@ -48,6 +48,5 @@ func readStub(name string) ([][]byte, error) {
 		msg += "data: " + string(data) + "\n"
 		msgs = append(msgs, []byte(msg))
 	}
-
```

Comment:
Nit-pick: keep the empty line before `return`

Reply:
Done.